src/ToolBox/superpmi/superpmi/commandline.cpp

```diff
@@ -637,3 +613,28 @@ bool CommandLine::Parse(int argc, char* argv[], /* OUT */ Options* o)
     }
     return true;
 }
+
+bool CommandLine::AddJitOption(int i, int argc, char ** argv, CommandLine::Options * o, LightWeightMap<DWORD, DWORD>** pJitOptions)
```

Comment:
Nits: formatting is off (e.g. chould be `char** argv`, not `char ** argv`) and `pJitOptions` should just be `jitOptions`.

Reply:
Yes, we do not run jit format for spmi sources. I run clang format manually occasionally.

Reply:
It is pJitOptions because it is pointer to pointer and we declare jitOptions inside the function.
I can't find a better solution if we need to initialize this pointer inside the function.